src/keys/apk.rs

```diff
@@ -58,17 +58,30 @@ impl From<&SecretKey> for APK {
 
 impl APK {
     /// Aggregate a set of [`PublicKey`] into the [`APK`].
```

Comment:
I suggest adding an `#Error` section to the docs of this function, explaining under what conditions this can fail.
```suggestion
    /// Aggregate a set of [`PublicKey`] into the [`APK`].
    ///
    /// # Errors
```